src/components/Match/Match.css

```diff
@@ -148,6 +159,117 @@
   vertical-align: middle;
 }
 
+.ward-log {
+  width: 100%;
+  font-size: .8em;
```

Comment:
I'll oblige but I think that using px as font size is a bad practice for mobility. I'd rather leave it that way or refactor to use em/rem all around.